### ISSUE TITLE
Feature request: negative margin

### ISSUE THREAD

**Issue by kevinlui1109** [vuetifyjs/vuetify#2313]
I have a thought about the negative margin

There are good margin spacing helpers defined in vuetify like mx-1, ma-1 but the negative margin
Negative margin can be defined like mt--3 which means margin top is -16px because mt-3 multiply -1.
So on mt--1 = -4px, mt--2 = -8px, mt--5 = -48px

```
<v-layout wrap>
    <v-flex xs12>
        <v-card></v-card>
    </v-flex>
</v-layout>
<v-layout wrap>
    <v-flex v-for="(item, i) in items" :key="i" mt-3 px-1 xs12 sm6 md4>
        <v-card></v-card>
    </v-flex>
</v-layout>
```

There are many ```<v-flex/>``` that are arranged with xs12, sm6 and md4, and the px-1 adds gutters between
```<v-flex/>``` but all of these are not aligned with the ```<v-flex xs12/>``` at the top. In this case, mx--1 is needed, so I want negative margin for the second ```<v-layout/>```.

I know that i can add px-1 on that xs12 v-flex

Thanks, tell me if there is another ways achieve it.

**Comment by donPuerto** [comment]
not only margin, negative values on padding us useful too...

**Comment by jacekkarczmarczyk** [comment]
@donPuerto https://stackoverflow.com/questions/4973988/why-does-css-not-support-negative-padding

**Comment by donPuerto** [comment]
@jacekkarczmarczyk ow, thanks for that info...

**Comment by johnleider** [comment]
Thank you for the feature request. Unfortunately, this is not something I would consider adding to the framework at this time. It, in my opinion, would be an extremely niche use-case and only add additional size to the styles.

**Comment by jacekkarczmarczyk** [comment]
That is implemented in vuetify 2.0
`<div class="ml-n3">`
